recipes/de2120-barcode-scanner/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "de2120-barcode-scanner" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/de2120_barcode_scanner-{{ version }}.tar.gz
+  sha256: 7d8e5fb88cd221213ea3b99493b0acdeb6d16655a1af393b466d4b4be66bf0e0
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+
+test:
+  imports:
+    - de2120_barcode_scanner
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://pypi.org/project/de2120-barcode-scanner/
+  summary: SparkFun Electronics DE2120 Barcode Scanner Breakout Package
+  license: MIT
+  license_file: LICENSE
```

Comment:
Can you include a link to the issue you opened upstream providing them with an example of how to include their LICENSE file in their pypi distribution:


https://github.com/psf/requests/blob/main/MANIFEST.in#L1
https://github.com/psf/requests/blob/main/setup.cfg#L2
They need to include
[metadata]
license_file = LICENSE





[17:45](https://ramonaoptics.slack.com/archives/D038AEV2QQ1/p1675118758602479)
from the setup.cfg
[17:46](https://ramonaoptics.slack.com/archives/D038AEV2QQ1/p1675118769882299)
and also have a [MANIFEST.in](http://manifest.in/) to add it to the "source distribution"
[17:46](https://ramonaoptics.slack.com/archives/D038AEV2QQ1/p1675118778727029)
the setup.cfg adds it to the "wheel" whl

Reply:
https://github.com/sparkfun/DE2120_Py/issues/1

Reply:
Can you add a reference to this issue in the recipe so we know to delete our copy of their license file when it is resolved.